plugin/plugin.go

```diff
@@ -0,0 +1,107 @@
+package plugin
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/spf13/viper"
+)
+
+func sendTo(method, url string, payload map[string]interface{}) (int, string, error) {
+	payloadJSON, err := json.Marshal(payload)
+	if err != nil {
+		return -1, "", err
+	}
+
+	var req *http.Request
+
+	if payload != nil {
+		req, err = http.NewRequest(method, url, bytes.NewBuffer(payloadJSON))
+		if err != nil {
+			return -1, "", err
+		}
+	} else {
+		req, err = http.NewRequest(method, url, nil)
+		if err != nil {
+			return -1, "", err
+		}
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	defer resp.Body.Close()
+	if err != nil {
+		return -1, "", err
+	}
+
+	body, _ := ioutil.ReadAll(resp.Body)
```

Comment:
Return error here (do not underline it). It is unlikely to occur an error here, but if it does we will have much headache to find it.